scripts/src/test/org/oppia/android/scripts/XMLSyntaxCheckTest.kt

```diff
@@ -0,0 +1,155 @@
+package org.oppia.android.scripts
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+/** Tests for [XMLSyntaxCheck]. */
+class XMLSyntaxCheckTest {
+  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
+  private val originalOut: PrintStream = java.lang.System.out
+
+  @Rule
+  @JvmField
+  public var tempFolder: TemporaryFolder = TemporaryFolder()
+
+  @Before
+  fun setUpTests() {
+    tempFolder.newFolder("testfiles")
+    java.lang.System.setOut(PrintStream(outContent))
+  }
+
+  @After
+  fun restoreStreams() {
+    java.lang.System.setOut(originalOut)
+  }
+
+  @Test
+  fun testXmlSyntax_validXML_xmlSyntaxIsCorrect() {
```

Comment:
``validXml``

Ditto elsewhere (make sure 'xml' is following expected camel case naming rules for all identifiers where it's used, including variables).

Reply:
Done.